build/annotate-clients.php

```diff
@@ -0,0 +1,43 @@
+<?php
+
+require __DIR__ . '/../vendor/autoload.php';
+
+$options = getopt('', ['class:', 'tag:', 'all']) + ['class' => [], 'tag' => []];
+
+array_walk($options, function (&$value) {
+    if (!is_array($value)) {
+        $value = [$value];
+    }
+});
+
+if (isset($options['all'])) {
+    $options['class'] = array_values(array_map(function (array $manifest) {
```

Comment:
What is this code doing? Are you supposed to pass in the classes to update as an argument?